CHANGELOG.md

```diff
@@ -1,6 +1,17 @@
 Change Log
 ==========
 
+# Version 3.8.5
+
+_2024-07-10_
+
+Version 3.8.5 is a maintenance release. New developments happen in the 4.x versions.
+
+* [all] Bump okio to 3.9.0 (#5868)
+* [runtime] Adding checks for json end_document in http transport (#5894)
+* [runtime] Adding checks for json end_document in http batching interceptors (#5892)
+* [codegen] Fix mapping of the builtin Float scalar type (#6047)
```

Comment:
I think there is https://github.com/apollographql/apollo-kotlin/pull/5975 as well?

Reply:
true! 👀

Reply:
Wait, sorry I meant #5972 I don't think we usually include docs changes in the changelog

Reply:
🤦 Right!